src/Razor/Microsoft.AspNetCore.Razor.Tools/src/GenerateCommand.cs

```diff
@@ -201,6 +201,7 @@ protected override bool ValidateArguments()
                 if (GenerateDeclaration.HasValue())
                 {
                     b.Features.Add(new SetSuppressPrimaryMethodBodyOptionFeature());
+                    b.Features.Add(new SuppressChecksumOptionsFeature());
```

Comment:
This prevents the pragma checksum from being generated on declaration files. The checksum is used by debugger. Declaration files aren't used as part of debugging, they're only used for component discovery. Having the checksum is actively harmful since any changes to the input would result in the checksum being different.